modules/cxx.rs/1.0.147/presubmit.yml

```
@@ -0,0 +1,15 @@
+matrix:
+  platform:
+    - macos_arm64
+    - ubuntu2404
+    - windows
+  bazel: [7.x, 8.x]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+      - '//...'
+    test_targets:
+      - '//...'
```

I think this is what is needed based on my understanding of 
[README.md#anonymous-module-test](https://github.com/bazelbuild/bazel-central-registry/blob/628c44a875e2de2ba096242b95df04bc2c57f3ac/docs/README.md#anonymous-module-test)

```suggestion
    build_targets:
      - '@cxx.rs//...'
    test_targets:
      - '@cxx.rs//...'
```

---

@dtolnay do you wanna try this suggestion and if that works I can open that pull-request?

---

- https://github.com/dtolnay/cxx/pull/1476

---

I had to validate something else anyway (https://github.com/dtolnay/cxx/pull/1475)